<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployeeInformation</class>
 <widget class="QMainWindow" name="EmployeeInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>511</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>872</width>
    <height>511</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>872</width>
    <height>511</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Employee Information</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color:qlineargradient(spread:pad, x1:0.435644, y1:0.767, x2:1, y2:0, stop:0 rgba(69, 20, 68, 255), stop:1 rgba(255, 255, 255, 255))}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="FirstName_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>First Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="LastName_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Last Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="MiddleName_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Middle Name:                                                          (if applicable)</string>
    </property>
   </widget>
   <widget class="QLabel" name="DOB_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>DOB(m/d/yyyy):</string>
    </property>
   </widget>
   <widget class="QLabel" name="StoreInformation_Label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>29</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Store Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="SSN_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>SSN:</string>
    </property>
   </widget>
   <widget class="QLabel" name="PersonalInformation_Label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Personal Information</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FirstName_input">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="MiddleName_Input">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LastName_Input">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SSN_Input">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDateEdit" name="DOB_Input">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>300</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2000</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
   </widget>
   <widget class="QComboBox" name="Department_Input">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Grocery</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pharmacy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bakery</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Costumer Service</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Front End</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Warehouse</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Department_Label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Department:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title_Label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Position/Title:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Title_Input">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PayInformation_Label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Pay Information</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>260</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Salary</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hourly</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Amount_Input">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>310</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial Rounded MT Bold&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Amount:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SUBMIT">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SUBMIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="FirstName_Error">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 4);
font: 11pt &quot;Arial Rounded MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LastName_Error">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 4);
font: 11pt &quot;Arial Rounded MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DOB_Error">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 4);
font: 11pt &quot;Arial Rounded MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SSN_ERROR">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 4);
font: 11pt &quot;Arial Rounded MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Amount_Error">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 4);
font: 11pt &quot;Arial Rounded MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="PayType_Error">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 4);
font: 11pt &quot;Arial Rounded MT Bold&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SUBMIT_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
